Libraries/LibWeb/Layout/TreeBuilder.h

```diff
@@ -64,6 +67,31 @@ class TreeBuilder {
     Vector<GC::Ref<Layout::NodeWithStyle>> m_ancestor_stack;
 
     u32 m_quote_nesting_level { 0 };
+
+    // Everything we need to recalculate `content` that contains `reversed` counters for a pseudo-element.
+    struct ElementWithReversedCounterContent {
+        GC::Ptr<DOM::Element> element;
+        CSS::PseudoElement pseudo_element;
+        GC::Ptr<DOM::Text> text;
+        u32 intial_quote_nesting_level;
+    };
+
+    AK::OwnPtr<AK::Vector<ElementWithReversedCounterContent>> m_content_with_reversed_counters_fixup_queue;
+
+    ALWAYS_INLINE AK::OwnPtr<AK::Vector<ElementWithReversedCounterContent>>& ensure_content_with_reverse_counter_fixup_queue()
```

Comment:
We know this always returns a value, so instead of worrying about OwnPtr, just return `Vector<ElementWithReversedCounterContent>&`.

You also don't need the `AK::` prefix above.